task/buildpacks-phases/0.1/buildpacks-phases.yaml

```diff
@@ -59,7 +59,7 @@ spec:
   steps:
     # Ensure the builder's user has read/write permissions for needed directories.
     - name: prepare
-      image: alpine
+      image: docker.io/library/alpine:3.12
```

Comment:
@vdemeester Since this image is available for various os architecture and each os architecture we have different digest, any particular architecture we need to choose or the default one i.e `amd64` and also is it fine to use this particular tag i.e `3.12`

Reply:
@PuneetPunamiya the manifest digest should be the same, so you "should" have only one digest. In any case, it's fine using the tag for this :wink: